### ISSUE TITLE
Migrate: Format output config to use prettier

### ISSUE THREAD

**Issue by TheLarkInn** [webpack/webpack-cli#70]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
Poorly formatted configs, or new ast nodes that are generated appear inline and hard to read after `--migrate`.

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
Be _prettier_ (put intended) than what is output now.

**If this is a feature request, what is motivation or use case for changing the behavior?**
I was able to add @jlongster's prettier and pass the string in to be formatted: 

Below is how I implemented this feature (at a very rough level). 

**transformations/index.js**

```javascript
  return pretty.format(ast.toSource(recastOptions), {singleQuote: true, tabWidth: 4});
```

### The results were, damn fine looking!

```javascript
var path = require('path');
var webpack = require('webpack');

// lets take this webpack v1 config and migrate it to v2 !!! <3333

module.exports = {
    debug: true,
    devtool: 'eval',
    entry: ['./src/index'],
    output: {
        path: path.join(__dirname, 'dist'),
        filename: 'index.js'
    },
    plugins: [
        new webpack.OccurenceOrderPlugin(),
        new webpack.optimize.UglifyJsPlugin({
            sourceMap: true
        }),
        new webpack.optimize.LoaderOptionsPlugin({
            debug: true,
            minimize: true
        })
    ],
    resolve: {
        modules: ['foo']
    },
    module: {
        rules: [
            {
                test: /\.js$/,
                use: 'babel-loader',
                include: path.join(__dirname, 'src')
            }
        ]
    }
};
```

Thoughts @pksjce @ev1stensberg @kenwheeler @okonet? Spoke with folks at prettier and they said if passing a direct string is a concern and overhead we could look into the future passing a direct AST.

**Comment by TheLarkInn** [comment]
Could technically be some low hanging fruit for someone to tackle also!!! Given we all agree we should format the ast before its printed to string.

**Comment by okonet** [comment]
I agree we should do that. My only concern is that it still not be a fit for many devs and keeping in mind prettier has almost zero configuration options we won't be able to please everyone. Which is okay for me :)

**Comment by TheLarkInn** [comment]
Yes, that was one of my concerns as well. I would assume that the most suspect ones though would be spaces and tabs which I think we could in the future accept args and then pass to prettier as args/opts

**Comment by pksjce** [comment]
We are anyway working around this [issue](https://github.com/benjamn/recast/issues/242). It would remove the need for https://github.com/webpack/webpack-cli/blob/master/package.json#L42 and thus enable us to use `yarn` on our cli :)

**Comment by ev1stensberg** [comment]
Closed per #207
